fabtests/man/fabtests.7.md

```diff
@@ -357,6 +357,14 @@ This will run "fi_rdm_atomic" for all atomic operations with
 	- 1024 bytes message size
 	- server node as 123.168.0.123
 
+## Run multinode tests
+
+	server and clients are invoked with the same command: 
+		fi_multinode -n <number of processes> -s <server_addr> 
+	
+	a process on the server must be started before any of the clients can be started 
+	succesfully. 
```

Comment:
For the multinode test, we want a more complete description of what this does.  See the ubertest description as more as a guide.  We want to explain what the test is doing in more detail.